src/main/java/bit/javaoop/Main.java

```diff
@@ -1,10 +1,20 @@
 package bit.javaoop;
 
-public class Main {
-    public static void main(String[] args) {
-        AbstractEmployee employee = new Employee("Garek Majęcki", 25_000);
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-        System.out.println(employee.getName() + " " + employee.getSalary());
+public class Main {
+    public static void main(String args[])
+    {
+        AbstractEmployee fixed = new FixedEmployeeNumberManager("Anna", 25_000f);
+        AbstractEmployee employee = new Employee("Jozef", 5_000f);
+        AbstractEmployee manager = new FixedEmployeeNumberManager("Jack", 5_000f);
+        fixed.addEmployee(employee);
+        //when
```

Comment:
You could clean this up a little bit :D

Those `given`, `when`, `then` comments should only be used in tests.